src/runtime/recipe/handle.ts

```diff
@@ -284,4 +284,8 @@ export class Handle {
 
     return result.join(' ');
   }
+
+  findInHandleConnection() {
+    return this._connections.find(conn => conn.direction === 'in');
```

Comment:
This is a good candidate for a slightly more general method, by passing the desired direction value in: `findConnectionByDirection(dir)`